src/SFA.DAS.EAS.Web/Views/EmployerManageApprentices/WhenToApplyChange.cshtml

```diff
@@ -21,102 +21,35 @@
             @Html.Partial("ValidationSummary", Html.ViewData.ModelState)
 
             <h1 class="heading-xlarge">@ViewBag.Title</h1>
-
-            <table class="apprentice-to-stop">
-                <thead>
-                    <tr>
-                        <th colspan="2" class="visually-hidden">Stop apprentice</th>
-                    </tr>
-                </thead>
-                <tbody>
-                    <tr>
-                        <td>Name</td>
-                        <td class="bold">@Model.Data.ApprenticeshipName</td>
-                    </tr>
-                    <tr>
-                        <td>Unique learner number</td>
-                        <td class="bold">@Model.Data.ApprenticeshipULN</td>
-                    </tr>
-                    <tr>
-                        <td>Apprenticeship training course</td>
-                        <td class="bold">@Model.Data.TrainingName</td>
-                    </tr>
-                    </tbody>
-                </table>
-
-
-
-
-
-                    <fieldset>
-                        <legend class="visually-hidden" aria-hidden="true">Select stop date</legend>
-                        <p class="strong">When did this apprentice stop?</p>
-                        <div class="form-group">
-                            <legend class="visually-hidden">Date of status change</legend>
-
-                            @Html.ValidationMessage("WhenToMakeChange", "Please choose an option", new { id = "error-message-WhenToMakeChange" })
-                            <label for=@($"WhenToMakeChange-{WhenToMakeChangeOptions.Immediately}") class="block-label selection-button-radio">
-                                @Html.RadioButton("WhenToMakeChange", WhenToMakeChangeOptions.Immediately, new { id = $"WhenToMakeChange-{WhenToMakeChangeOptions.Immediately}", dataOptionName = $"{WhenToMakeChangeOptions.Immediately}", onclick = $"sfa.tagHelper.radioButtonClick('{ViewBag.Title}', '{WhenToMakeChangeOptions.Immediately}')" })
-                                Today
+            <fieldset>
+                <legend class="visually-hidden" aria-hidden="true">Select stop date</legend>
+                <p>The earliest date you can enter is this apprenticeship's start date of @Model.Data.ApprenticeStartDate.ToGdsFormatWithoutDayAbbrMonth().  Any outstanding payments will be made for activity up to this date.</p>
```

Comment:
Looking at the (very pixellated!) screen shot attached to the story, the example given is '... start date of June 2018...', so we should use `ToGdsFormatWithoutDay`, rather than `ToGdsFormatWithoutDayAbbrMonth`.

Reply:
Good spot